src/vs/workbench/contrib/mergeEditor/browser/view/mergeEditor.ts

```diff
@@ -520,28 +520,6 @@ export class MergeEditorResolverContribution extends Disposable {
 	) {
 		super();
 
-		const editorInputFactory: EditorInputFactoryFunction = (editor) => {
-			return {
-				editor: instantiationService.createInstance(
-					MergeEditorInput,
-					editor.resource,
-					{
-						uri: editor.resource,
-						title: '',
-						description: '',
-						detail: ''
-					},
-					{
-						uri: editor.resource,
-						title: '',
-						description: '',
-						detail: ''
-					},
-					editor.resource
-				)
-			};
-		};
-
 		const mergeEditorInputFactory: MergeEditorInputFactoryFunction = (mergeEditor: IResourceMergeEditorInput): EditorInputWithOptions => {
```

Comment:
I wonder if the factory should use `mergeEditor.input1.label` and `mergeEditor.input1.description` if provided for the merge editor title and description. Same for `input2`.

Reply:
I pushed that.